browser/components/newtab/tests/xpcshell/test_RemoteNewTabLocation.js

```diff
@@ -0,0 +1,39 @@
+/* globals Assert, RemoteNewTabLocation, Services */
+"use strict";
+
+Components.utils.import("resource:///modules/RemoteNewTabLocation.jsm");
+Components.utils.import("resource://gre/modules/Services.jsm");
+Components.utils.importGlobalProperties(["URL"]);
+
+add_task(function* () {
+  let defaultHref = RemoteNewTabLocation.href;
+
+  Assert.ok(RemoteNewTabLocation.href, "Default location has an href");
```

Comment:
Nit: I think we are are supposed to avoid using `Assert.` https://developer.mozilla.org/en/docs/Mochitest